protocol/v2/ssv/runner/voluntary_exit.go

```diff
@@ -90,6 +93,12 @@ func (r *VoluntaryExitRunner) ProcessPreConsensus(logger *zap.Logger, signedMsg
 		return errors.Wrap(err, "could not submit voluntary exit")
 	}
 
+	logger.Debug("voluntary exit submitted successfully",
```

Comment:
@moshe-blox, @lior-blox  this will probably need to be added to spec-alignment.. let me know who you want to do it.

Reply:
@y0sher done, it was just a metric line